### ISSUE TITLE
[ BUG ] 2.6 - Context buttons duplicate when contexts are re-ordered in the tree

### ISSUE THREAD

**Issue by jonleverrier** [modxcms/revolution#13666]
### Summary
When there are more than 1 contexts setup with Revolution 2.6, after re-ordering them in the tree, the "New Document" and "Refresh" buttons are duplicated

### Step to reproduce
1. Install a new version of 2.6
2. Add another context
3. Re-order the context by dragging the context before the Website context in the tree
4. Move the context back to its original position (after the Website context)

### Observed behavior
Duplicate "New Document" and "Refresh" buttons are added to the newly created context.
![screen shot 2017-11-03 at 13 44 11](https://user-images.githubusercontent.com/2373940/32374320-64ae9830-c09d-11e7-885b-ddbcb6ac082b.png)

### Expected behavior
No duplication of buttons should appear

### Environment
MODX version 2.6

**Comment by mindeffects** [comment]
Sorry, can't confirm that. Which is a good thing?
I duplicate the "web" context to "web2" and moved them around in "FF 57.0b12 (64-bit)" and "Chromium 59.0.3071.115 (64-Bit)". No dupe buttons with MODX 2.6.0-pl.

**Comment by jonleverrier** [comment]
That is a good thing @mindeffects !

I've uploaded a video just to confirm what i'm doing/seeing (sorry for the bad quality and watermark):
https://www.dropbox.com/s/briulvuznsvhc7p/contextbug.mp4?dl=0

I can see this on Firefox 55.0.3 and Safari Version 10.1.2 (10603.3.8), in addition to Chrome v61.0.31

**Comment by Ruslan-Aleev** [comment]
Yes, the problem is relevant, also when dragging the context in the "preview" (next to the cursor) all context icons with controls are drawn in "preview".

![context_dragging](https://user-images.githubusercontent.com/12523676/53095905-5b22ee00-3537-11e9-9d91-1391c80d199b.gif)
